### DISCUSSION TITLE
bad response on Iphone

### DISCUSSION THREAD

**Discussion by remmob** [ExperienceLovelace/ha-floorplan#483] [Q&A]
I have made clickable icons, but the are rather small and only the actual part of the SVG icon is touchable/clickable.
To solve the problem, I have put a big transparent square on top the icon.  to make the click area bigger, gave it its own id.

This works fine on PC with mouse, but is not working on Iphone.
I'm a doing something wrong?

**Comment by remmob** [comment]
I have beter responce on chrome, first i was testing safari.

**Reply by remmob** [reply]
In the companion app there seems to be no problem, only when I use Safari on Iphone.

**Reply by OzGav** [reply]
Probably a Safari problem although it is surprising It works on chrome as iOS devices use the same underlying engine for all browsers
